packages/Portal/src/Portal.spec.tsx

```diff
@@ -14,12 +14,36 @@ describe('packages/Portal', () => {
     </div>,
   );
 
+  function isElement(el: Node): el is HTMLElement {
```

Comment:
Will be nice once we move this to `lib`.

Reply:
I guess I should make the same `| null` improvement here that I did in the other branch. Definitely looking forward to having a shared util :)

Reply:
Indeed